pages/Javascript-API.md

```diff
@@ -44,6 +44,39 @@ Each object contains sub-objects which correlate to each element's subelements a
 
 If changes are made to the `config` object, you must call `scene.updateConfig()` for them to take effect.
 
+####`scene.config.textures`
```

Comment:
Almost everything else on the other non-overview pages concern the structure and syntax of the scene file specifically – I'd been using this as a place to put anything lower-level accessed through JS… which are also all about accessing and modifying JS objects, no?